examples/opensora_hpcai/README.md

```diff
@@ -249,6 +254,10 @@ video_folder/part01/vid001.mp4,a cartoon character is walking through
 video_folder/part01/vid002.mp4,a red and white ball with an angry look on its face
 ```
 
+> [!IMPORTANT]
+> OpenSora v1.1 also requires the `length` field, which represents the number of frames,
```

Comment:
can you add instructions on how to get the `length` for the original text-video data csv?